### DISCUSSION TITLE
ofxtools purpose

### DISCUSSION THREAD

**Discussion by Capt-Ahab726** [csingley/ofxtools#121] [Q&A]
I noticed that your software was recently updated to work with USAA's most recent change. Do you know if the software is able to successfully connect to the OFX server since they also require you to pass the client ID in addition to the Access ID and PIN. Also, will your software work as a standalone app as I'm using Microsoft Money which is only compatible with the OFX standard. The developer of a Python script called PocketSense, doesn't seem very keen on making the necessary changes to his program as he would most likely have to create a subroutine just for the additional strings and special format needed to successfully connect to USAA's new server.

**Comment by csingley** [comment]
At least one user has managed to connect to USAA.

It's a good idea to read the `README`, which is prominently displayed on the project's front page for your convenience.  Within you'll find links to more documentation.
